tests/xmltester/tests/robust/overlay/TestOverlay-shapely-1061.xml

```diff
@@ -0,0 +1,18 @@
+<run>
+<precisionModel type="FLOATING" />
+
+<case>
+<desc>
+  Failure in intersection of two geometries in GEOS < 3.9.0, from Shapely report.
```

Comment:
Annoying, but this errors out for me because `<` is a special character.

Reply:
That is the one annoyance of XML.  Use amp-lt-semicolon, or reword to "in GEOS 3.8"

Reply:
Ah, tests passed locally, but easy enough to update!